samples/testapp/src/commonMain/kotlin/org/multipaz/testapp/provisioning/ProvisioningSupport.kt

```diff
@@ -0,0 +1,231 @@
+package org.multipaz.testapp.provisioning
+
+import io.ktor.http.Url
+import io.ktor.util.encodeBase64
+import kotlinx.coroutines.channels.Channel
+import kotlinx.coroutines.channels.SendChannel
+import kotlinx.coroutines.sync.Mutex
+import kotlinx.coroutines.sync.withLock
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.JsonArray
+import kotlinx.serialization.json.add
+import kotlinx.serialization.json.buildJsonArray
+import kotlinx.serialization.json.buildJsonObject
+import kotlinx.serialization.json.jsonObject
+import kotlinx.serialization.json.jsonPrimitive
+import kotlinx.serialization.json.put
+import org.multipaz.asn1.OID
+import org.multipaz.crypto.Algorithm
+import org.multipaz.crypto.Crypto
+import org.multipaz.crypto.EcPrivateKey
+import org.multipaz.crypto.X509Cert
+import org.multipaz.provision.openid4vci.OpenID4VCIBackend
+import org.multipaz.provision.openid4vci.OpenID4VCIClientPreferences
+import org.multipaz.securearea.KeyAttestation
+import org.multipaz.util.toBase64Url
+import kotlin.random.Random
+import kotlin.time.Clock
+import kotlin.time.Duration.Companion.minutes
+import kotlin.time.Duration.Companion.seconds
+
+class ProvisioningSupport: OpenID4VCIBackend {
```

Comment:
Add comment explaining that in a real app, this would be done server-side and we're only doing it wallet-side here to avoid a dependency on a backend?